src/Models/BaseElement.php

```diff
@@ -310,6 +361,53 @@ public function getCMSFields()
                 $fields->fieldByName('Root.History')
                     ->addExtraClass('elemental-block__history-tab tab--history-viewer');
             }
+
+            // Viewer groups (remove tabs from silverstripe/asset-admin)
+            $fields->removeByName('ViewerGroups');
+            $fields->removeByName('EditorGroups');
+            $fields->removeByName('CanEditType');
+
+            //group options field
+            $viewersOptionsField = OptionsetField::create(
+                "CanViewType",
+                _t(__class__ . '.ACCESSHEADER', "Who can view this element?")
+            );
+
+            //group options set
+            $viewersOptionsSource = [
+                InheritedPermissions::INHERIT => _t(
+                    __class__ . '.ACCESSINHERIT',
```

Comment:
Can you please switch the magic constants for `__CLASS__`?

Reply:
No problem, done.